src/core/Track.cpp

```diff
@@ -2190,7 +2371,18 @@ void TrackOperationsWidget::mousePressEvent( QMouseEvent * me )
 void TrackOperationsWidget::paintEvent( QPaintEvent * pe )
 {
 	QPainter p( this );
-	p.fillRect( rect(), palette().brush(QPalette::Background) );
+	
+	if( hasColor && ! m_trackView->getTrack()->getMutedModel()->value() ) 
+	{
+		QRect coloredRect( 0, 0, 10, m_trackView->getTrack()->getHeight() );
+		
+		p.fillRect( rect(), palette().brush(QPalette::Background) );
```

Comment:
This line can be factored out above the if statement, then the else block can be removed.